sdk/python/foundation-models/system/inference/text-to-image/safe-text-to-image-batch-deployment.ipynb

```diff
@@ -0,0 +1,693 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Azure AI Content Safety (AACS) enabled text-to-image batch endpoint (Preview)\n",
```

Comment:
What do you think about framing it as "Create an Azure AI Content Safety (AACS) enabled text-to-image batch endpoint (Preview) .." ?
similar to other notebooks we have